src/otx/algorithms/detection/configs/rotated_detection/configuration.yaml

```diff
@@ -350,28 +350,6 @@ algo_backend:
       type: UI_RULES
     visible_in_ui: false
     warning: null
-  storage_cache_scheme:
```

Comment:
It's just a question. Why is it removed?

Reply:
Thought it's irrelevant, but I've reverted. Thanks.